### PR TITLE
Add November 2015 TC39 meeting notes.

### LINKED ISSUE #36: What about notes from November TC39 meeting?
More than a month have passed.

### PR DESCRIPTION
**Pull request by ljharb**
Fixes #36.

Happy to fix any markdown formatting issues.

### CODE REVIEW

**Comment by rwaldron** [comment]
Every line that marks the beginning of a person saying a thing needs a blank line before it, otherwise the markdown looks like this: https://github.com/ljharb/tc39-notes/blob/2015-11/es7/2015-11/nov-17.md

File: es7/2015-11/nov-17.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+BT: time for a regexp talk ?
+
+#### Conclusion/Resolution
+None yet.
+
+
+## Object.values/entries
+
+JHD: OK for stage 3? It got signoff
+<general consensus>
```
Comment (rwaldron) [inline es7/2015-11/nov-17.md]: Don't use `<` `>`—that's "html"

**Comment by ljharb** [comment]
ah, icky. will fix and update, thanks

File: es7/2015-11/nov-17.md (hunk 1 above)
Comment (ljharb) [inline es7/2015-11/nov-17.md]: on it

**Comment by ljharb** [comment]
(i'll fix the third markdown file tonight)

**Comment by ljharb** [comment]
k, should be all updated

File: es7/2015-11/nov-18.md [hunk 2]
```diff
@@ -0,0 +1,1016 @@
...
+
+YK: should we remove it from map
+
+DD: lets remove it from everything, and put it on `Function.prototype['@@species'] = this;`
+
+AWB: it was added for array instance extensibility.
+
+DD: that would break zepto
+
+DE: Object.__proto__ is Function.prototype, which have a species returning this, which breaks the web.
```
Comment (azu) [inline es7/2015-11/nov-18.md]: should be 

    `Object.__proto__`

File: es7/2015-11/nov-18.md [hunk 3]
```diff
@@ -0,0 +1,1016 @@
...
+
+KS: it seems like the cascade of missing species will break things.
+
+DE: we could erase species create, is the species of the constructor object, if it is fallback. Then species does what I wants to do.
+
+YK: I believe Function.prototype behind right about Object.__proto__ is unfortunate.
```
Comment (azu) [inline es7/2015-11/nov-18.md]: ditto

File: es7/2015-11/nov-18.md (hunk 2 above)
Comment (ljharb) [inline es7/2015-11/nov-18.md]: fixed, thanks

**Pull request merged**